{"uid":"39fe3860224df38c","name":"用户注册登录删除-预期失败","fullName":"testcases.scenario_test.test_03_register_login_delete.TestRegLogDelete#test_user_register_login_delete_fail","historyId":"7164a98ce91bd9f479af2fa0b0f95820","time":{"start":1744708531543,"stop":1744708531616,"duration":73},"description":"该用例是针对 注册-登录-删除 场景的测试","descriptionHtml":"<p>该用例是针对 注册-登录-删除 场景的测试</p>\n","status":"skipped","statusMessage":"AssertionError: 接口返回码是 【 2005 】, 返回信息：手机号已被注册！！！ \nassert False is True\n +  where False = <core.result_base.ResultBase object at 0x10412a070>.success","statusTrace":"self = <testcases.scenario_test.test_03_register_login_delete.TestRegLogDelete object at 0x104187d90>\ntestcase_data = {'address': '深圳市宝安区', 'admin_pwd': '123456', 'admin_user': 'wintest4', 'except_code': 3004, ...}\n\n    @allure.story(\"用例--注册/登录/删除--预期失败\")\n    @allure.description(\"该用例是针对 注册-登录-删除 场景的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"用户注册登录删除-预期失败\")\n    @pytest.mark.negative\n    @pytest.mark.xfail(reason=\"xfail示例：预期失败的用例，如存在尚未解决的Bug等\")\n    @pytest.mark.usefixtures(\"delete_register_user\")\n    def test_user_register_login_delete_fail(self, testcase_data):\n        username = testcase_data[\"username\"]\n        password = testcase_data[\"password\"]\n        telephone = testcase_data[\"telephone\"]\n        sex = testcase_data[\"sex\"]\n        address = testcase_data[\"address\"]\n        admin_user = testcase_data[\"admin_user\"]\n        admin_pwd = testcase_data[\"admin_pwd\"]\n        except_result = testcase_data[\"except_result\"]\n        except_code = testcase_data[\"except_code\"]\n        except_msg = testcase_data[\"except_msg\"]\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = register_user(username, password, telephone, sex, address)\n        step_1(username, password, telephone, sex, address)\n>       assert result.success is True, result.error\nE       AssertionError: 接口返回码是 【 2005 】, 返回信息：手机号已被注册！！！ \nE       assert False is True\nE        +  where False = <core.result_base.ResultBase object at 0x10412a070>.success\n\ntestcases/scenario_test/test_03_register_login_delete.py:87: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"delete_register_user","time":{"start":1744708531460,"stop":1744708531542,"duration":82},"status":"passed","steps":[{"name":"前置步骤 ==>> 清理数据","time":{"start":1744708531541,"stop":1744708531542,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false}],"attachments":[],"parameters":[],"stepsCount":1,"hasContent":true,"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false},{"name":"testcase_data","time":{"start":1744708531542,"stop":1744708531542,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false}],"testStage":{"description":"该用例是针对 注册-登录-删除 场景的测试","status":"skipped","statusMessage":"AssertionError: 接口返回码是 【 2005 】, 返回信息：手机号已被注册！！！ \nassert False is True\n +  where False = <core.result_base.ResultBase object at 0x10412a070>.success","statusTrace":"self = <testcases.scenario_test.test_03_register_login_delete.TestRegLogDelete object at 0x104187d90>\ntestcase_data = {'address': '深圳市宝安区', 'admin_pwd': '123456', 'admin_user': 'wintest4', 'except_code': 3004, ...}\n\n    @allure.story(\"用例--注册/登录/删除--预期失败\")\n    @allure.description(\"该用例是针对 注册-登录-删除 场景的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"用户注册登录删除-预期失败\")\n    @pytest.mark.negative\n    @pytest.mark.xfail(reason=\"xfail示例：预期失败的用例，如存在尚未解决的Bug等\")\n    @pytest.mark.usefixtures(\"delete_register_user\")\n    def test_user_register_login_delete_fail(self, testcase_data):\n        username = testcase_data[\"username\"]\n        password = testcase_data[\"password\"]\n        telephone = testcase_data[\"telephone\"]\n        sex = testcase_data[\"sex\"]\n        address = testcase_data[\"address\"]\n        admin_user = testcase_data[\"admin_user\"]\n        admin_pwd = testcase_data[\"admin_pwd\"]\n        except_result = testcase_data[\"except_result\"]\n        except_code = testcase_data[\"except_code\"]\n        except_msg = testcase_data[\"except_msg\"]\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = register_user(username, password, telephone, sex, address)\n        step_1(username, password, telephone, sex, address)\n>       assert result.success is True, result.error\nE       AssertionError: 接口返回码是 【 2005 】, 返回信息：手机号已被注册！！！ \nE       assert False is True\nE        +  where False = <core.result_base.ResultBase object at 0x10412a070>.success\n\ntestcases/scenario_test/test_03_register_login_delete.py:87: AssertionError","steps":[{"name":"步骤1 ==>> 注册用户","time":{"start":1744708531616,"stop":1744708531616,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"username","value":"'测试test'"},{"name":"password","value":"'123456'"},{"name":"telephone","value":"'13599999999'"},{"name":"sex","value":"'1'"},{"name":"address","value":"'深圳市宝安区'"}],"stepsCount":0,"hasContent":true,"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false}],"attachments":[{"uid":"ca625963e39777af","name":"log","source":"ca625963e39777af.txt","type":"text/plain","size":1645},{"uid":"1fc8b1933793a8a","name":"stderr","source":"1fc8b1933793a8a.txt","type":"text/plain","size":2024}],"parameters":[],"stepsCount":1,"hasContent":true,"shouldDisplayMessage":true,"attachmentsCount":2,"attachmentStep":false},"afterStages":[{"name":"delete_register_user::0","time":{"start":1744708531623,"stop":1744708531705,"duration":82},"status":"passed","steps":[{"name":"后置步骤 ==>> 清理数据","time":{"start":1744708531705,"stop":1744708531705,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false}],"attachments":[],"parameters":[],"stepsCount":1,"hasContent":true,"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false}],"labels":[{"name":"story","value":"用例--注册/登录/删除--预期失败"},{"name":"feature","value":"场景：用户注册-用户登录-删除用户"},{"name":"epic","value":"针对业务场景的测试"},{"name":"severity","value":"minor"},{"name":"tag","value":"negative"},{"name":"parentSuite","value":"testcases.scenario_test"},{"name":"suite","value":"test_03_register_login_delete"},{"name":"subSuite","value":"TestRegLogDelete"},{"name":"host","value":"192.168.10.5"},{"name":"thread","value":"24590-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"testcases.scenario_test.test_03_register_login_delete"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[{"name":"点击，跳转到对应用例的链接地址","url":"https://www.cnblogs.com/wintest","type":"test_case"},{"name":"点击，跳转到对应BUG的链接地址","url":"https://www.cnblogs.com/wintest","type":"issue"}],"hidden":false,"retry":false,"extra":{"severity":"minor","retries":[],"categories":[],"tags":["negative"]},"source":"39fe3860224df38c.json","parameterValues":[]}